tests/common/errors.py

```diff
@@ -23,4 +17,12 @@ def __init__(self, msg, results=None):
         self.results = results
 
     def __str__(self):
-        return "{}\nAnsible Results => {}".format(self.message, dump_ansible_results(self.results))
```

Comment:
Maybe better will be to have only "repr" function defined? Its according to

https://docs.python.org/2.7/reference/datamodel.html#object.__repr__

"The return value must be a string object. If a class defines repr() but not str(), then repr() is also used when an “informal” string representation of instances of that class is required."

Reply:
I mean to move code from "str" to "repr"

Reply:
Or maybe better to create separate common method, which will be called from "__repr__" and from "__str__" without doing calls like "__repr__" -> "__str__"?

Reply:
Good idea! Updated.